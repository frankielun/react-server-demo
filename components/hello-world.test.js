import React from 'react';
import HelloWorld from './hello-world';
import renderer from 'react-test-renderer';

it('renders correctly2', () => {
	const tree = renderer
		.create(<HelloWorld />)
		.toJSON();

	expect(tree).toMatchSnapshot();
});
