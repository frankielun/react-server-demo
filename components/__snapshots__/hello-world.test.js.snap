// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly2 1`] = `
<div>
  <h2>
    Hello, Worlddddd
    
  </h2>
  <button
    onClick={[Function]}
  >
    Get More Excited!
  </button>
</div>
`;
